Adding contributors

KatherineMichel opened the pull request:
Installed all-contributors-cli and started adding contributors

jefftriplett commented:
@KatherineMichel this looks great. Would you mind adding info for how to run / update the contributors to the README?

KatherineMichel commented:
Sure! I was planning to do that, but hadn't gotten to it yet. Will do it.

KatherineMichel commented:
I realized while writing the steps that for most people it would be easiest to run the commands locally without installing globally. I had already done a basic init and installed the developer dependency. I added the scripts to run locally and added instructions to README. Can add global info if needed, but may be moot.
